spec/acceptance/sites_spec.rb

```diff
@@ -44,48 +46,72 @@ def sites_body_params
     sites_project_id_param
     let(:project_id) { project.id }
     let(:authentication_token) { admin_token }
-    standard_request_options(:get, 'INDEX (as admin)', :ok, {expected_json_path: 'data/0/custom_latitude', data_item_count: 1})
+    standard_request_options(:get, 'INDEX (as admin)', :ok, expected_json_path: 'data/0/custom_latitude', data_item_count: 1)
   end
 
   get '/projects/:project_id/sites' do
     sites_project_id_param
     let(:project_id) { project.id }
     let(:authentication_token) { owner_token }
-    standard_request_options(:get, 'INDEX (as owner)', :ok, {expected_json_path: 'data/0/custom_latitude', data_item_count: 1})
+    standard_request_options(:get, 'INDEX (as owner)', :ok, expected_json_path: 'data/0/custom_latitude', data_item_count: 1)
   end
 
   get '/projects/:project_id/sites' do
     sites_project_id_param
     let(:project_id) { project.id }
     let(:authentication_token) { writer_token }
-    standard_request_options(:get, 'INDEX (as writer)', :ok, {expected_json_path: 'data/0/custom_latitude', data_item_count: 1})
+    standard_request_options(:get, 'INDEX (as writer)', :ok, expected_json_path: 'data/0/custom_latitude', data_item_count: 1)
   end
 
   get '/projects/:project_id/sites' do
     sites_project_id_param
     let(:project_id) { project.id }
     let(:authentication_token) { reader_token }
-    standard_request_options(:get, 'INDEX (as reader)', :ok, {expected_json_path: 'data/0/custom_latitude', data_item_count: 1})
+    standard_request_options(:get, 'INDEX (as reader)', :ok, expected_json_path: 'data/0/custom_latitude', data_item_count: 1)
+  end
+
+  Array[
+    'id',
+    'name',
+    'description',
+    'creator_id',
+    'updater_id',
+    'created_at',
+    'updated_at',
+    'project_ids',
+    'location_obfuscated',
+    'custom_latitude',
+    'custom_longitude',
+    'timezone_information',
+    'description_html',
+    'image_urls'
+  ].each do |x|
```

Comment:
this is re-running the test 13 times.... 

`expected_json_path` can take an array of paths. Redefine this list as a variable of expected paths and pass that into `expected_json_path` and return this test to only running once